src/test/test_ep_cred_common.h

```diff
@@ -118,6 +131,11 @@ struct crt_proto_rpc_format my_proto_rpc_fmt_0[] = {
 		.prf_req_fmt	= &CQF_ping,
 		.prf_hdlr	= ping_hdlr_0,
 		.prf_co_ops	= NULL,
+	}, {
+		.prf_flags	= CRT_RPC_FEAT_QUEUE_FRONT,
+		.prf_req_fmt	= &CQF_ping,
+		.prf_hdlr	= ping_hdlr_1,
+		.prf_co_ops	= NULL,
```

Comment:
Same question about version mismatches if that is ever an issue. Should this add on to the end of the structure. More of a question.

Reply:
Adding onto the end doesn't solve the problem because the protocol incorporates the size so the new RPC wouldn't be useable.  You need to change the protocol version, and then the offsets doesn't matter (as long as you update everything correctly).

Good practice would be to add onto the end however.